scripts/stujo_migrate_gcp.sh

```
@@ -0,0 +1,104 @@
+export STRATO_SSH_PASS SSH_ASKPASS="$AP" SSH_ASKPASS_REQUIRE=force
+SSH_OPTS="-o StrictHostKeyChecking=accept-new -o ConnectTimeout=20"
+
+echo "==> Rsyncing Rails public/system (logos + job PDFs) from prod"
+mkdir -p "${FILES_ROOT}/system"
+setsid -w rsync -a --info=progress2 -e "ssh ${SSH_OPTS}" \
+  "${STRATO_SSH_HOST}:/home/edu/stujo/public/system/logos" \
+  "${STRATO_SSH_HOST}:/home/edu/stujo/public/system/jobs" \
+  "${FILES_ROOT}/system/" </dev/null
+
+echo "==> Opening SSH tunnel to prod MySQL (13306 -> 127.0.0.1:3306)"
+setsid -w ssh ${SSH_OPTS} -N -L 13306:127.0.0.1:3306 -o ExitOnForwardFailure=yes \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)